opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -246,7 +249,10 @@ impl<R: RuntimeChannel> SpanProcessor for BatchSpanProcessor<R> {
         let result = self.message_sender.try_send(BatchMessage::ExportSpan(span));
 
         if let Err(err) = result {
-            global::handle_error(TraceError::Other(err.into()));
+            otel_error!(
```

Comment:
will this be triggered when channel is full? if yes, we shouldn't do such logging as that can overwhelm internal logs

Reply:
yes it will. Will make it debug in that case.

Reply:
updated as below:

```rust
        let result = self.message_sender.try_send(BatchMessage::ExportSpan(span));

        if let Err(err) = result {
            otel_debug!(
                name: "BatchSpanProcessor.OnEnd.SendResultError",
                reason = format!("{:?}", TraceError::Other(err.into()))
            );
        }
```